containers/release-candidate/daffodil-release-candidate

```diff
@@ -170,6 +165,8 @@ sbt \
   "daffodil-japi/genjavadoc:doc" \
   "daffodil-sapi/doc" \
 
+echo "Removing old release candidates..."
+svn delete --force $DAFFODIL_DIST_DIR/$VERSION-* 2> /dev/null
```

Comment:
All this piping errors to dev null seems very problematic to me. Under maintenance if one goofs up the pushd/popd pairings,  or goof up the definitions of things like $DAFFODIL_DIST_DIR you won't get any diagnostics.  At least not from these commands that will fail. So long as that's not a problem (because the errors would be obvious elsewhere in the script) then I'm ok with it I guess. 

If the script is mostly straight line code, then there's little once can mess up under maintenance, so my concern may be unwarranted. 

Is there a less fragile way to do this? E.g., just redirect to a temp file,.....

Reply:
Yeah, I was being a little lazy here. The reason for the /dev/null redirect here is that there won't be an existing release candidate if this is the first one. So svn will output an error that there's nothing to delete. I just wanted to hide that, but you're right that this could potentially hide other real errors.

It's probably a better idea to check for the existence before deleting. I'll make the update.